in_toto/in_toto_sign.py

```diff
@@ -51,6 +55,11 @@
   # Verify Layout signed with three keys
   in-toto-sign -f root.layout -k pub_key0 pub_key1 pub_key2 --verify
 
+  # Sign layout with default gpg key in default gpg keyring
+  in-toto-sign -f gpg.layout --gpg
+
+  # Verify layout with a gpg key found in keyring
+  in-toto-sign -f gpg.layout --gpg 3BF8135765A07E21BD12BF89A5627F6BF439F3C2
```

Comment:
Sign layout and Verify layout are the same?  This is odd.

Reply:
Oh, I forgot the `--verify` parameter. Good eyes!